modules/serverless-rn-1-10-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-10-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.10.0
+
+[id="new-features-1-10-0_{context}"]
+== New features
+* The annotation `knative-eventing-injection=enabled` that was previously used to annotate namespaces for broker creation is now deprecated. The new annotation is `eventing.knative.dev/injection=enable`. For more information, see the documentation on _Creating a broker as a cluster administrator using namespace annotation_.
+* Multi-container support is now available on Knative as a Technology Preview feature. You can enable multi-container support in the `config-features` config map.
+// For more information see...... need to add docs on this / link to upstream docs?
+
+[id="fixed-issues-1-10-0_{context}"]
+== Fixed issues
+* In previous releases, Knative Serving had a fixed, minimum CPU request of 25m for `queue-proxy`. If your cluster had any value set that conflicted with this, for example, if you had set a minimum CPU request for `defaultRequest` of more than `25m`, the Knative Service failed to deploy. This issue is fixed in 1.10.0.
+
+[id="known-issues-1-10-0_{context}"]
+== Known issues
```

Comment:
IMO, this issue should be documented as known: https://issues.redhat.com/browse/SRVKS-641